server/router.js

```diff
@@ -0,0 +1,41 @@
+const axios = require('axios');
+const router = require('express').Router();
+const config = require('../config');
+
+const atelierUrl = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/';
+const authToken = config.token;
```

Comment:
I had to move my config.js file and make sure the key matched, but otherwise all testing via postman worked.